codemod/spotlight_util.py

```diff
@@ -0,0 +1,34 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2007-2008 Facebook
+
+from Cocoa import *
+
+
+class SpotlightUtil(object):
+    """
+    Searches a directory hierarchy using OS/X spotlight
+    """
+
+    @staticmethod
```

Comment:
Sure!  Can do that, I wanted it conditional on whether we were on Darwin, though it looks like Cocoa isn't always available (I'll comment on that comment of yours)